test/helpers/starlightVersions.ts

```diff
@@ -1,15 +1,15 @@
 export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_BALANCES = [1300];
 export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_PROXY = [1300];
-export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_TREASURY = [1300, 1301, 1302, 1303, 1400];
-export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_CONTAINER_REGISTRAR = [1300, 1301, 1302, 1303, 1400];
-export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_SERVICES_PAYMENT = [1300, 1301, 1302, 1303, 1400];
+export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_TREASURY = [1300, 1301, 1302, 1303, 1400, 1500];
+export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_CONTAINER_REGISTRAR = [1300, 1301, 1302, 1303, 1400, 1500];
+export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_SERVICES_PAYMENT = [1300, 1301, 1302, 1303, 1400, 1500];
 export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_ETH_CLIENT = [1300];
 export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_ETH_TOKEN_TRANSFERS = [1300, 1301];
-export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_DATA_PRESERVERS = [1300, 1301, 1302, 1303, 1400];
+export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_DATA_PRESERVERS = [1300, 1301, 1302, 1303, 1400, 1500];
 export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_IDENTITY = [1300];
 export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_POOLED_STAKING = [1300, 1301, 1302, 1303];
-export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_XCM = [1300, 1301, 1302, 1303, 1400];
+export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_XCM = [1300, 1301, 1302, 1303, 1400, 1500];
 // TODO: Remove 1400 from exclusion list as soon as ForeignAssetsCreator pallet is integrated in 1400
-export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_FOREIGN_ASSETS_CREATOR = [1300, 1301, 1302, 1303, 1400];
+export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_FOREIGN_ASSETS_CREATOR = [1300, 1301, 1302, 1303, 1400, 1500];
 // TODO: Remove 1400 from exclusion list as soon as inactivity tracking pallet is integrated in 1400
-export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_INACTIVITY_TRACKING = [1300, 1301, 1302, 1303, 1400];
+export const STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_INACTIVITY_TRACKING = [1300, 1301, 1302, 1303, 1400, 1500];
```

Comment:
I guess we remove the 1500 as soon as we enable the disabled pallets in starlight in master